CHANGELOG.md

```diff
@@ -3,6 +3,10 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [4.15.0] - 2020-10-23
+## Changed
+- Modified `update_amentities()` and `update_amenity_properties()` to follow correct PATCH functionality and not send items to be updated that are null.
+
```

Comment:
Is there ever a scenario where a user might want to intentionally `null` a field?

Reply:
There might be, but I would suggest an empty string would probably be a better identifier of remove a field than a None in this case.

Reply:
That seems reasonable